Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Dan Levin <dan@badpacket.in>, Martin Häcker <spamfaenger@gmx.de>"
+LABEL description="This image is intended as a developent environment for task-tracker."
+
+ENV LC_ALL=C.UTF-8 LANG=C.UTF-8  
+
+# How does docker figure out when to rerun these lines?
+# FIXME according to https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache
```

Comment:
The way that I solve this problem is to regularly trigger cache cleaning on my system. I always call `docker image prune --filter until=12h` after every build as a post_build hook which clears out all build cache layers created prior to 12 hours from the current build. This ensures that within any 12 hour window, I will be able to benefit from docker image build cache layers that were performed in that window, but that the cache is also regularly invalidated and system package installation operations are re-run.

Reply:
maybe there's a better solution that leverages a COPY operation or something in the docker file itself... but I haven't come up with a better idea yet.